### DISCUSSION TITLE
Dev docs list version as 1.12; 1.11 not yet released

### DISCUSSION THREAD

**Discussion by bertiewooster** [sympy/sympy#23792] [General]
The docs dev version https://docs.sympy.org/dev/index.html lists the version as 1.12 at the top of the page. I'm guessing this is because the 1.11 branch was created (though not released yet). The dev version is correctly listed as 1.11 in the left navbar on the latest release docs version https://docs.sympy.org/latest/index.html.

![image](https://user-images.githubusercontent.com/1335522/179379397-6666a0ef-de66-4b91-84ce-ed27136ba5ed.png)

**Comment by oscarbenjamin** [comment]
The dev docs are built from master which now does represent the future docs for 1.12 so it's correct to say 1.12 there. The sidebar gets updated at the final stage of release:
https://github.com/sympy/sympy/blob/2f606ba2ccb1d9e43d7ae1bce97bc26677ff5592/release/update_docs.py#L61-L66
